### DISCUSSION TITLE
Where is the VSCode Extension Issue tracker?

### DISCUSSION THREAD

**Discussion by FrostKiwi** [community/community#36081] [Copilot Conversations] [answered]
A recent update broke Copilot for me, when behind a company's https proxy by removing access to the retry connection button.

Posting in [microsoft/vscode](https://github.com/microsoft/vscode) got my issue closed for being extension related.
The discussion page here seems to be the wrong place to post(?) since I'm not actually making a discussion or question, but want an issue tracked somehow: https://github.com/orgs/community/discussions/35263

So where *do* you track issues? The marketplace page links to the discussion page" https://marketplace.visualstudio.com/items?itemName=GitHub.copilot  but the discussions are not specific to the extension, so I'm am I just polluting the discussion page with extension specific issues?

**Comment by iamcjwilliams** [comment, marked as answer]
@FrostKiwi The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). I will create an internal issue of your previous discussion post and make sure our copilot team provides an update and I'll give you an update within this discussion
